pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -335,28 +247,8 @@ func (t ResolvedPipelineTask) hasRunObjectsStarted() bool {
 	return false
 }
 
-// isConditionStatusFalse returns true when any tasks or a singular task has succeeded condition with
```

Comment:
I'm not totally sure I understand why this function is being removed, since similar functionality is used in pipelinerunstate.go. Maybe it just needs to be renamed? I think what I find confusing about it is that condition status applies to a single taskrun or run, but a resolvedpipelinetask can have many. I would call it `isFailure` but we already have an `isFailure` function-- I'm not sure why `isFailure` checks anything other than condition status = false. Let me look into whether `isFailure` could have a better name.

Reply:
@lbernick I renamed this function and split it into `areTaskRunsConditionStatusFalse()` and `areRunObjectsConditionStatusFalse()` since it applies to multiple runs.

https://github.com/tektoncd/pipeline/blob/2ab7287416e17caa48591325bacf63d392961405/pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go#L250-L266